vendor/assets/stylesheets/normalize.css

```diff
@@ -0,0 +1,419 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
i don't think there will be any more third-party css (if i have any say in it 😄); we could have installed it via npm and then added a script to copy the file into our vendor file but it didn't seem worth it for this relatively static file, but i think that should be the protocol for libs that may need updating over time